packages/yoga/src/Checkbox/CheckboxSwitch.theme.js

```diff
@@ -36,15 +36,31 @@ const CheckboxSwitch = ({
       backgroundColor: colors.primary,
     },
     disabled: {
-      backgroundColor: colors.elements.lineAndBorders,
+      backgroundColor: colors.elements.white,
     },
   },
   focus: {
     checked: {
-      backgroundColor: hexToRgb(colors.primary, 0.2),
+      backgroundColor: hexToRgb(colors.text.secondary, 0.5),
     },
     disabled: {
-      backgroundColor: hexToRgb(colors.elements.selectionAndIcons, 0.2),
+      backgroundColor: hexToRgb(colors.elements.lineAndBorders, 0.25),
```

Comment:
According to figma, this color should be `colors.elements.backgroundAndDisabled` and the opacity should be `0.5`
```suggestion
      backgroundColor: hexToRgb(colors.elements.backgroundAndDisabled, 0.5),
```